### ISSUE TITLE
"make install" to install libocxl to /usr/lib

### ISSUE THREAD

**Issue by luyong6** [OpenCAPI/libocxl#23]
Hi @deece 
After 'make install', the libocxl.so was copied to /user/local/lib64. But I still need to set `LD_LIBRARY_PATH=.....` to make an opencapi AFU run.  It is not a default path, at least in my ubuntu18.04 it seems not. 

```
luyong@zzfp142p1:~/libocxl$ sudo make install
rm -rf docs
	[DOCS-MAN]
cd docs/man/man3 && \
	ls | grep -vi ocxl | xargs rm
	[DOCS-HTML]
mkdir -p /usr/local/lib64
mkdir -p /usr/local/include
mkdir -p /usr/local/share/man/man3
mkdir -p /usr/local/share/doc/libocxl/search
install -p -m 0755 obj/libocxl.so.1.1 /usr/local/lib64/
ln -s libocxl.so.1.1 /usr/local/lib64/libocxl.so.1
ln -s libocxl.so.1.1 /usr/local/lib64/libocxl.so
install -p -m 0644 src/include/libocxl.h  /usr/local/include/
install -p -m 0644 -D docs/man/man3/* /usr/local/share/man/man3
install -p -m 0644 -D docs/html/*.* /usr/local/share/doc/libocxl
install -p -m 0644 -D docs/html/search/* /usr/local/share/doc/libocxl/search
```

**Comment by deece** [comment]
That's right, you can also add an entry in /etc/ld.so.conf.d/ if you want to use your locally built version. The packaged version lives in /usr (https://launchpad.net/ubuntu/bionic/+source/libocxl).

We intentionally install to /usr/local to avoid clobbering files that are maintained by the package manager.

If you do want to override this behavior, you can pass the PREFIX, libdir, etc (see Makefile.rules) variable overrides like so: `make PREFIX=/usr install`

**Comment by luyong6** [comment]
So what's the suggested way? 
`sudo apt-get install libocxl-dev`? 
Would you put it on README on homepage?

**Comment by deece** [comment]
Yup, unless you have a reason for running the dev version.

We'll consider listing distro install instructions, thanks for the suggestion.

That reminds me, @fbarrat, Ubuntu Bionic is on 1.0.0, while Cosmic is on 1.1.0. There are a number of bug fixes in 1.1.0, so we should get Bionic updated if possible.

**Comment by deece** [comment]
Added binary install instructions to README.md: 8ea4b9abe7e2e547fb8e7fb27abb75ce5deebce4
